src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+
+
+namespace System.Reflection.Emit.Experimental
+{
+    class AssemblyBuilder: System.Reflection.Assembly
+    {
+        internal AssemblyBuilder() { }
+
+        public void Save(string assemblyFileName)
+        {
+//New method
+        }
+
+        public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access) { throw new NotImplementedException("Under construction"); }// code forthecoming
```

Comment:
Once you fill in the implementation, you will need to change this to return your `System.Reflection.Emit.Experimental.AssemblyBuilder` instead of the system assembly builder. 

(Since this is just a scaffolding, it is ok keep it as is.)